src/domain/card/CardNumber/CardNumber.stories.tsx

```
@@ -0,0 +1,94 @@
+    return (
+      <CardNumber
+        cardNumberRefs={cardNumberRefs}
+        cardNumber={cardNumber}
+        cardNumberErrorMessage={cardNumberErrorMessage}
+        handleCardNumberInputChange={handleCardNumberInputChange}
+      />
+    );
+  },
+};
+
+export const Error = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)